arm-sql/2014-04-01/swagger/dataMasking.json

```diff
@@ -0,0 +1,460 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Azure Sql Database API spec",
+        "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases.",
+        "version": "2014-04-01"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/dataMaskingPolicies/Default": {
+            "put": {
+                "tags": [
+                    "DataMasking"
+                ],
+                "operationId": "DataMasking_CreateOrUpdatePolicy",
```

Comment:
I would prefer to keep all the DataMasking APIs under the same operation group rather than giving each its own group.

---
In reply to: [80145048](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80145048) [](ancestors = 80145048)